wolfcrypt/src/aes.c

```diff
@@ -12109,19 +12109,19 @@ static WARN_UNUSED_RESULT int wc_AesFeedbackCFB1(
             pt = (byte*)aes->reg;
 
             /* LSB + CAT */
-            tmp = (0X01 << bit) & in[0];
-            tmp = tmp >> bit;
+            tmp = (byte)((0X01U << bit) & in[0]);
```

Comment:
dunno why this one wasn't resolved with the rest -- same explanation as the rest, with the outermost cast necessary to avoid warning, due to implicit promotion of the operands to `&`.